soccer_vision_msgs/msg/MarkingIntersection.msg

```diff
@@ -0,0 +1,26 @@
+# Describes a detected marking intersection from vision modules, in image coordinates.
+# An intersection is defined as a point where two line markings intersect.
+# The term ray is used to describe the outgoing rays from the center of the
+# intersection.
+#
+# For example, looking down at a perfectly upright T-intersection on the ground from above, you would have:
+# - num_rays = 3
+# - heading_rays = [0, pi / 2, pi]
+
+# A bounding box that surrounds the intersection in the image, this only has to include
+# the parts where the markings are intersecting, and not the rays that extend out.
+vision_msgs/BoundingBox2D bb
```

Comment:
But how much of the intersection should be shown?

Like this:
![image](https://user-images.githubusercontent.com/15075613/134703441-6895d0c6-5f81-4aa8-9177-2906c09d86bd.png)

or this:
![image](https://user-images.githubusercontent.com/15075613/134703733-7840c591-61a0-4043-aae3-74b8fb9cf04f.png)

Reply:
I think this comes down to the question, who's going to use the bounding box of the intersection?

I can't think of something that will use the bounding box of an intersection markings, apart from visualization tools.

Perhaps, we don't need a bounding box for intersections either.

Reply:
Me neither, for TORSO-21 we labeled them as Keypoints (only x and y) and our YOLO predicts a bounding box, because of the architecture, but the width and height are fixed, so it learns to keep them at a constant 5% image width or so. I would suggest using 2d coordinates for this. We also don't need a theta as in the vision message pose2d.

Reply:
Remove the bounding box field